tests/testthat/test-parse.R

```diff
@@ -125,7 +150,10 @@ test_that("parse_connect_rfc3339() handles fractional seconds", {
 })
 
 test_that("make_timestamp produces expected output", {
-  withr::defer(Sys.setenv(TZ = Sys.getenv("TZ")))
+  original_tz <- Sys.getenv("TZ")
```

Comment:
Maybe leave a comment above this saying that the test itself will mutate the time zone a few times, and we want to reset?

You could also wrap each block with `withr::with_envvar()`

Reply:
`local_envvar()` everywhere :)